.tekton/basic-component-pipeline.yaml

```diff
@@ -206,10 +206,10 @@ spec:
     taskRef:
       name: output-image-tag-construction
 
-  - name: build-container
+  - name: build-container-amd64
     params:
     - name: IMAGE
-      value: $(params.output-image-repo):$(tasks.output-image-tag-construction.results.FINAL_IMAGE_TAG)
```

Comment:
In the mean time, operator-bundle builds arrived and they bring `output-image-tag-construction` thingy. See https://github.com/stackrox/stackrox/pull/12037/files#diff-852bc59fb24fb28fe17c4ecfb8332be7dbe6476cc8a52c888cba481b2c2aeb99R198

You should use `$(params.output-image-repo):$(tasks.output-image-tag-construction.results.FINAL_IMAGE_TAG)-amd64` here and in similar places for arch-specific images, not `tasks.determine-image-tag.results.IMAGE_TAG`.

----

TIL that operator-bundle being `FROM scratch`-based container with only data still comes for each arch and with a manifest.

```json
$ podman manifest inspect registry.redhat.io/advanced-cluster-security/rhacs-operator-bundle:4.5.0    
{
    "schemaVersion": 2,
    "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
    "manifests": [
        {
            "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
            "size": 426,
            "digest": "sha256:afe3518921c5f14f11f7543f384c6cfc32eaae86d2a38bb6e36b783aeec53f79",
            "platform": {
                "architecture": "amd64",
                "os": "linux"
            }
        },
        {
            "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
            "size": 426,
            "digest": "sha256:b235dddbd9b87d565ccd2366ad615a825736c9f8126da62dfec1895c032a5a37",
            "platform": {
                "architecture": "ppc64le",
                "os": "linux"
            }
        },
        {
            "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
            "size": 426,
            "digest": "sha256:465631cade0b59189d12075ece6c4e1e18653fa61c99fa1c5c4bf8adaecc832b",
            "platform": {
                "architecture": "s390x",
                "os": "linux"
            }
        }
    ]
}
```